app/models/profile.rb

```diff
@@ -1,15 +1,10 @@
 class Profile < ApplicationRecord
   belongs_to :user
 
-  validates :data, presence: true
```

Comment:
This validation no longer makes sense, since `data` defaults to `{}`. Previously we'd always have a `custom_attributes` key but that is gone now.